can/interface.py

```diff
@@ -27,21 +23,12 @@ def __new__(cls, other, channel=None, *args, **kwargs):
             can.rc['interface'] = kwargs['bustype']
```

Comment:
I've been thinking about this. Wouldn't be more logical to never change can.rc if a custom bustype has been given? One might think that the default interface and channel should always be used when nothing else has been specified when initiating the Bus, but now it depends on the last created bus.